src/internal/puck/components/ColorSelector.tsx

```diff
@@ -1,57 +1,39 @@
-import React, { useState, useCallback } from "react";
-import { Field, FieldLabel } from "@measured/puck";
+import React, { useState } from "react";
+import { FieldLabel } from "@measured/puck";
 import { RenderProps } from "../../utils/renderEntityFields.ts";
 import { Color, ColorResult, SketchPicker } from "react-color";
 
-export type ColorSelectorProps = {
-  label: string;
-};
-
-export const ColorSelector = (props: ColorSelectorProps): Field => {
-  const colorSelectorRenderer = useCallback(
-    ({ field, value, onChange }: RenderProps) => {
-      const [isOpen, setIsOpen] = useState(false);
+export const ColorSelector = ({ field, value, onChange }: RenderProps) => {
```

Comment:
Does this change how someone needs to use the component now since it's returning something different? Ideally a user can just use it straight up in their config without needing to defined the 'custom' stuff themselves, though it's not the end of the world if that's the case.

Reply:
The user shouldn't ever need to use this component directly. When the user specifies `type: "color"` in the `theme.config.ts`, we render this component in the sidebar in `src/internal/utils/constructThemePuckFields.ts`